DIGITAL_OCEAN_README.md

```diff
@@ -0,0 +1,62 @@
+# Digital Ocean Setup
+
+### Gotchas that will save you time:
+
+* Create a project in Google Cloud Provider (GCP). Copy the project ID for the next step.
+* Create a project in Digital Ocean with the name of the ID that you copied in the GCP project. e.g. `My-Project-1234567`
+  - Under the hood the Terraform code that is generated uses the `projectId` field in your `config/main.json` to store
+    the Terraform state in a GCP bucket. This same field is also used to access the Digital Ocean project when building your infrastructure so these names MUST be
+    the same.
+* The ssh user name only works with `root` when digitalocean is the provider. Any other name will fail to fully install.
+* Inside of your `config/main.json` the `machineType` field must have one of the slug values found in the [DigitalOcean docs](https://slugs.do-api.dev/). At the time of this writing `s-4vcpu-8gb-amd` is more than enough (~$48/mo.)
+
+### Example config
+
+Found in `config/main.json`
```

Comment:
This is already available in `config/main.sample.json` and I'd remove it from here.